test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPushCommandTest.cs

```diff
@@ -336,13 +338,6 @@ public void PushCommand_PushToServerBasicAuth()
                     using (var server = new MockServer())
                     {
                         server.Listener.AuthenticationSchemes = AuthenticationSchemes.Basic;
-                        server.Get.Add("/nuget", r =>
```

Comment:
Like i mentioned, we do not send out the "get" like nuget2 does to resolve the redirect link which has been removed by nuget server, so we have no request to record